src/styles/frontend-shared.scss

```diff
@@ -33,32 +33,6 @@
   }
 }
 
-// Used in the AdderToolbar, a transparent button that has an icon and a small
```

Comment:
The only use of this pattern was the `AdderToolbar` buttons. These are now handled with utility styles. If we _reuse_ this pattern I will externalize it.